### PR TITLE
Add [joyGetPosEx]

### PR DESCRIPTION
**Pull request by thehighestmath**

### CODE REVIEW

**Comment by myCrack** [comment]
There should be two new modules: `winmm.mpl` and `winmmPrivate.mpl`. These modules should contain the code you added.

File: windows/winmm.mpl [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# It is forbidden to use the content or any part of it for any purpose without explicit permission from the owner.
+# By contributing to the repository, contributors acknowledge that ownership of their work transfers to the owner.
+
+"control.Natx" use
+"winmmPrivate" use
+
+winmmInternal: {
+  JOYINFOEX: @JOYINFOEX;
+
+  getJoyPosEx: @joyGetPosEx;
```
Comment (myCrack) [inline windows/winmm.mpl:14]: There should be comment, like this https://github.com/Matway/mpl-sl/blob/e53da6eb03aad8a4021bdc870c4c0a613fad9923/windows/kernel32.mpl#L59 one.

**Review by myCrack — CHANGES_REQUESTED** [review]

File: windows/winmm.mpl (hunk 1 above)
Comment (thehighestmath) [reply windows/winmm.mpl:15]: fixed

File: windows/winmm.mpl [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2021 Matway Burkow
+#
+# This repository and all its contents belong to Matway Burkow (referred here and below as "the owner").
+# The content is for demonstration purposes only.
+# It is forbidden to use the content or any part of it for any purpose without explicit permission from the owner.
+# By contributing to the repository, contributors acknowledge that ownership of their work transfers to the owner.
+
+"control.Natx" use
```
Comment (myCrack) [inline windows/winmm.mpl:8]: Delete the line.

File: windows/winmm.mpl [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+# It is forbidden to use the content or any part of it for any purpose without explicit permission from the owner.
+# By contributing to the repository, contributors acknowledge that ownership of their work transfers to the owner.
+
+"winmmPrivate" use
+
+winmmInternal: {
+  JOYINFOEX: @JOYINFOEX;
+
+  # winmm.Lib should be included for these functions 
+  getJoyPosEx: @joyGetPosEx;
```
Comment (myCrack) [inline windows/winmm.mpl:14]: Rename [getJoyPosEx] to [joyGetPosEx].

**Review by myCrack — APPROVED** [review]
Thank you.

**Review by myCrack — APPROVED** [review]

**Pull request merged**